cookbook/how_to_upgrade_to_new_version.rst

```diff
@@ -0,0 +1,79 @@
+.. index::
+    single: Upgrade
+
+How to upgrade to new version
+==============================
+
+1. Checkout from the GitHub Repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To retrieve new version and upgrade your ORO CRM instance, please execute the next steps:
```

Comment:
Edit: To retrieve a new version and upgrade your ORO CRM instance, please execute the following steps: